tools/accuracy_checker/configs/efficientnet-b0-tf.yml

```diff
@@ -0,0 +1,30 @@
+models:
+  - name: efficientnet-b0
+
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   public/efficientnet-b0-tf/FP32/efficientnet-b0-tf.xml
```

Comment:
if it is possible, please add scoring model on source framework.
AC support usage checkpoint meta now and I tested this feature on this model, so i know that it works :)